packages/mos-gateway/src/mosHandler.ts

```diff
@@ -482,6 +488,11 @@ export class MosHandler {
 		deviceOptions.primary.heartbeatInterval =
 			deviceOptions.primary.heartbeatInterval || DEFAULT_MOS_HEARTBEAT_INTERVAL
 
+		if (deviceOptions.secondary?.id && this._openMediaHotStandby[deviceOptions.secondary.id]) {
+			//@ts-expect-error  this is not yet added to the official mos-connection
+			deviceOptions.secondary.openMediaHotStandby = true
```

Comment:
Does this mean that this PR is not mergable yet? Because there needs to be an update to mos-connection before?

Reply:
Yes, it appears so.
The PR https://github.com/nrkno/sofie-mos-connection/pull/105 has been merged in that repository, but v4.2.0 has only been half created, so cannot be referenced here yet.

Reply:
https://github.com/nrkno/sofie-mos-connection/pull/105 Is this the PR?

Reply:
That was the PR, but there are some additional fixes without a PR yet